account_multicurrency_revaluation/demo/currency_demo.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record model="res.currency.rate" id="currency_rate_usd_2017_01">
+        <field name="name">2017-01-15</field>
+        <field name="rate">2</field>
+        <field name="currency_id" ref="base.USD"/>
+    </record>
+    <record model="res.currency.rate" id="currency_rate_usd_2017_02">
+        <field name="name">2017-02-15</field>
+        <field name="rate">4</field>
+        <field name="currency_id" ref="base.USD"/>
+    </record>
+    <record model="res.currency.rate" id="currency_rate_usd_2017_03">
+        <field name="name">2017-03-15</field>
+        <field name="rate">2.5</field>
+        <field name="currency_id" ref="base.USD"/>
+    </record>
+    <record model="res.currency.rate" id="currency_rate_gbp_2017_01">
+        <field name="name">2017-01-15</field>
+        <field name="rate">2</field>
+        <field name="currency_id" ref="base.GBP"/>
+    </record>
+    <record model="res.currency.rate" id="currency_rate_gbp_2017_02">
+        <field name="name">2017-02-15</field>
+        <field name="rate">4</field>
+        <field name="currency_id" ref="base.GBP"/>
+    </record>
+    <record model="res.currency.rate" id="currency_rate_gbp_2017_03">
+        <field name="name">2017-03-15</field>
```

Comment:
Could you change all the names so the year is automatically the actual one ? Otherwise it's a pain to test it manually.
Test case would have to be adapted as well.